test/games/strategy/engine/framework/mapDownload/MapDownloadPropertiesTest.java

```diff
@@ -0,0 +1,42 @@
+package games.strategy.engine.framework.mapDownload;
```

Comment:
Test of the property reader we now have. We write to a config file and then read it back using MapDownloadProperties, making sure we get the same property value that we wrote.